webpack-test.config.js

```diff
@@ -25,6 +25,7 @@ const babel = {
 
 module.exports = {
     entry,
+    mode: 'production',
```

Comment:
Is this needed? Even if deployed, this remains a testing-only file.

Reply:
Ah. It's not strictly necessary here. What it does is that it allows for various `production` environment only optimization, which includes minimizer (UglifyJS). But since we don't use it for test, we can leave it out for now.

Reply:
I'll look into the node requirements and windows environment. Missed that since my local machine has a later version.